packages/roosterjs-editor-api/lib/table/editTable.ts

```diff
@@ -16,9 +16,37 @@ export default function editTable(editor: Editor, operation: TableOperation) {
             vtable.writeBack();
             editor.focus();
 
-            if (!editor.select(start, end)) {
-                editor.select(editor.contains(td) ? td : vtable.getCurrentTd());
-            }
+            let cellToSelect = CalculateCellToSelect(operation, vtable.row, vtable.col);
+            editor.select(
+                vtable.getCell(cellToSelect.newRow, cellToSelect.newCol).td,
+                PositionType.Begin
+            );
         }, ChangeSource.Format);
     }
 }
+
+function CalculateCellToSelect(operation: TableOperation, currentRow: number, currentCol: number) {

```

Comment:
Nit: Rename to `calculateCellToSelect`. In typescript/javascript we use camel casing for function names.